src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write your code here
+const items = document.querySelectorAll('.population');
+
+const population = [...items].map((item) => parseInt(item.textContent.replace(/,/g, '')));
+const totalValue = population.reduce((prev, curr) => prev + curr);
+const averageValue = totalValue/ population.length;
+
+const total = document.querySelector('.total-population');
+const average = document.querySelector('.average-population');
+
+total.textContent = value(totalValue);
+average.textContent = value(averageValue);
+
+function value(number) {
```

Comment:
```suggestion
function getStringifiedPopulation(number) {
```
value не подходящее название для функции